R/zzz.R

```diff
@@ -3,10 +3,8 @@
   if (!interactive()) return()
   base::packageStartupMessage(
     "  <\")))><  The goldfish package in R\n\n",
-    pkgname, ": version ", utils::packageVersion("goldfish"),
-    ", \"Butterfly Tail\", created on ",
-    utils::packageDescription("goldfish", fields = "Date"),
```

Comment:
The date is useful for users, no?

Reply:
Done in [efb815f](https://github.com/snlab-ch/goldfish/pull/77/commits/efb815fde17ae6643e98bc282790cef0bd725438)